mappings/net/minecraft/entity/projectile/FishingBobberEntity.mapping

```diff
@@ -27,6 +28,11 @@ CLASS net/minecraft/class_1536 net/minecraft/entity/projectile/FishingBobberEnti
 		ARG 2 owner
 		ARG 3 lureLevel
 		ARG 4 luckOfTheSeaLevel
+	METHOD method_26086 isOpenOrWaterAround (Lnet/minecraft/class_2338;)Z
+		ARG 1 pos
+	METHOD method_26087 isOpenOrWater (Lnet/minecraft/class_2338;)Z
```

Comment:
What does this mean?

Reply:
Returns whether this block is air or water/waterlogged.

Reply:
This is called by `isOpenOrWaterAround` in a blockpos stream in a 5 * 3 * 5 area to ensure in this area there is no artificial blocks. Looks like the fix for fishing machine.